mgmtsystem_review/readme/CONTRIBUTORS.md

```diff
@@ -0,0 +1,15 @@
+- Maxime Chambreuil \<<maxime.chambreuil@savoirfairelinux.com>\>
+- Loïc Faure-Lacroix \<<loic.lacroix@savoirfairelinux.com>\>
+- Gervais Naoussi \<<gervaisnaoussi@gmail.com>\>
+- Luk Vermeylen \<<luk@allmas-it.be>\>
+- Eugen Don \<<eugen.don@don-systems.de>\>
+- Jose Maria Alzaga \<<jose.alzaga@aselcis.com>\>
+- Julien Coux \<<julien.coux@camptocamp.com>\>
+- [Tecnativa](https://www.tecnativa.com):
+  - Ernesto Tejeda
+
+Trobz
+
+- Dung Tran \<<dungtd@trobz.com>\>
+- Yvan Dotet \<<yvan.dotet@logicasoft.eu>\>
```

Comment:
Yvan Dotet is not part of Trobz, we should move him out of Trobz's contributor block.
btw, on this occasion let's revise our Trobz block to the same format as the other companies above.
example: - [Trobz](https://www.trobz.com):